src/modules/colorPicker/ColorPickerUI/ColorPickerUI.csproj

```diff
@@ -6,6 +6,7 @@
     <OutputPath>$(SolutionDir)$(Platform)\$(Configuration)\modules\ColorPicker</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <GenerateSatelliteAssembliesForCore>true</GenerateSatelliteAssembliesForCore>

```

Comment:
should we add it as part of Directory.props file instead?

Reply:
Plus, directory.props doesn't propagate for publishing directly from the csproj, so it would still need it in the project files for projects that are published.